overview.qmd

```diff
@@ -21,4 +21,34 @@ We will not use the course page on Blackboard. Instead, all course materials can
 Questions regarding personal circumstances may be shared through e-mail with the teaching team. Please don't use e-mail for other questions unless it cannot be avoided - we'd prefer if you ask us in class or open an issue in the course repo. 
 
 ## Syllabus
-This course website serves as syllabus. Background information, e.g. about course aims, can be found on [Osiris](https://osiris-student.uu.nl/#/onderwijscatalogus/extern/cursus?cursuscode=202000010&taal=en&collegejaar=huidig)
\ No newline at end of file
+This course website serves as syllabus. Background information, e.g. about course aims, can be found on [Osiris](https://osiris-student.uu.nl/#/onderwijscatalogus/extern/cursus?cursuscode=202000010&taal=en&collegejaar=huidig)
+
+## Teaching methods
+
+The learning activities in this course include lectures/seminars, supervised labs with exercises/tutorials, and throughout the course students will work on a portfolio with deliverables. 
+
+The topics discussed in this course are:
+
+1  Markup languages
```

Comment:
This information is also on the index page. May be worth pointing there rather than reproducing the information here